### ISSUE TITLE
Error on GNOME 40: ReferenceError: setSecondsEnabledGNOME is not defined

### ISSUE THREAD

**Issue by browseria** [injcristianrojas/UTCClock#15]
This was working fine for the last few weeks until I upgraded to v20 from extensions.gnome.org. Now I get:
```
sudo journalctl -f /usr/bin/gnome-shell
Apr 05 11:52:53 fedora34 gnome-shell[1861]: [UTCClock-20] enabling...
Apr 05 11:52:53 fedora34 gnome-shell[1861]: JS ERROR: Extension utcclock@injcristianrojas.github.com: ReferenceError: setSecondsEnabledGNOME is not defined
                                            enable@/home/browseria/.local/share/gnome-shell/extensions/utcclock@injcristianrojas.github.com/extension.js:78:5
                                            _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:167:32
                                            loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:350:26
                                            callback@resource:///org/gnome/shell/ui/extensionDownloader.js:222:39
                                            gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:106:13
```

**Comment by injcristianrojas** [comment]
Thank you very much for the report. I've just uploaded a new version to Extensions. In the meantime, while it's getting approved and if you're feeling adventurous, check out the [class](https://github.com/injcristianrojas/UTCClock/tree/class) branch and install manually. This beta version doesn't include the preferences dialog, all options have been moved to a dropdown in the clock itself.

If you don't feel adventurous, and you don't want for Extensions approval, just go to [master](https://github.com/injcristianrojas/UTCClock) and do a manual install.
